ouroboros-consensus/ouroboros-consensus-test-infra/src/Test/ThreadNet/General.hs

```diff
@@ -225,7 +225,6 @@ runTestNetwork TestConfig
           { nodeInfo
           , mkRekeyM
           } = mkTestConfigMB
-    setCurrentTime dawnOfTime
```

Comment:
Not sure I understand this. How come this is redundant?

Reply:
`setCurrentTime dawnOfTime` also occurs five lines up in the same computation.

Reply:
Ah :)